### PR TITLE
Add menu for mobile version

### PR DESCRIPTION
**Pull request by benwmx**
## Changes ⬆️ 

* Add new classes to the menu.
* Add a closing button to the menu.
* Add a js function for opening the menu when the menu button is clicked.
* Add a js function for closing the menu when the menu button is clicked.
* style the menu items.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- [ ] Change function keywords to arrow functions following the Javascript best practices.
- [ ] Update the readme to include `Getting started` instructions.

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: menu.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    menuItem[item].classList.add('menu-item');
+  }
+  menu.classList.add('mobile-menu');
+  document.querySelector('nav > img').classList.toggle('visible');
+  document.querySelector('nav > div').classList.toggle('visible');
+}
+
+function closeMenu() {
+  const menu = document.querySelector('nav>ul');
+  const menuItem = Array.from(document.querySelectorAll('nav > ul li'));
```
Comment (mikenath223) [inline menu.js]: - [ ] Kindly change these regular functions to `arrow functions`. In the project requirements particularly [this section](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.), it encourages the use of ES6. Javascript's ES6 brings new syntax and awesome features to make your code more modern and readable. It allows you to write less code and do more. ES6 introduces us to many great features like arrow functions. The arrow function is really awesome and makes your code more readable, more structured, and look like modern code. Take a minute and also go through [this article on why ES6 arrow functions are so awesome](https://zendev.com/2018/10/01/javascript-arrow-functions-how-why-when.html).

File: index.html [hunk 2]
```diff
@@ -179,8 +200,10 @@ <h1 class="title">Professional Art <br>Printing Data</h1>
         <div class="about-me">
             <h1> About Me</h1>
             <p>
-                Hello I’m a software developer! I can help you build a product , feature or website Look through some of
-                my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                Hello I’m a software developer! I can help you build a
```
Comment (mikenath223) [inline index.html:203]: - [ ] In the readme.md file, please include a `Getting started` section with instructions on how a user can clone and run the project.

**Comment by benwmx** [comment]
thank you for your review @mikenath223  I did all the requested changes.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**